lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -1491,8 +1491,10 @@ public function TicketChange($ticketid, array $props)
             $props['source'] = $ticket['source'];
         }
 
-        if ($ticket['priority'] != $props['priority'] && isset($props['priority'])) {
-            $notes[] = trans('Ticket\'s priority has been changed from $a to $b.', $RT_PRIORITIES[$ticket['priority']], $RT_PRIORITIES[$props['priority']]);
+        if ($ticket['priority'] != $props['priority']) {
+            $a = (is_null($ticket['priority']) || !isset($ticket['priority'])) ? trans("unset") : $RT_PRIORITIES[$ticket['priority']];
+            $b = is_null($props['priority']) ? $RT_PRIORITIES[$props['priority']] : trans("unset");
```

Comment:
Tutaj chyba to samo?
Przypominam, że isset() dla zmienej o wartości `null` zwróci `false`.

Reply:
W przypadku $a takie uproszczenie jest możliwe bo zawsze istnieje $ticket['priority']
W przypadku $b już nie bo nie zawsze istnieje zmienna $props['priority'] - ale wymyśliłem inny pojedyńczy test e64d126.

Reply:
@chilek tak będzie już wystarczająco?